server_extensions_extra/src/main/resources/batm-extensions.xml

```diff
@@ -1990,11 +1991,11 @@
             <param name="password" />
             <param name="host" />
             <param name="port" />
-            <cryptocurrency>SPICE</cryptocurrency>
```

Comment:
Do not remove SPICE token, add a new line with your token

Reply:
The latest commit addresses this!